tracer/src/Datadog.Trace/ClrProfiler/ScopeFactory.cs

```diff
@@ -109,7 +109,11 @@ internal static Span CreateInactiveOutboundHttpSpan(Tracer tracer, string httpMe
                 span = tracer.StartSpan(OperationName, tags, serviceName: serviceName, traceId: traceId, spanId: spanId, startTime: startTime, addToTraceContext: addToTraceContext);
 
                 span.Type = SpanTypes.Http;
+#if NET6_0_OR_GREATER
+                span.ResourceName = string.Create(null, stackalloc char[16 + resourceUrl?.Length ?? 0], $"{httpMethod} {resourceUrl}");
```

Comment:
@tonyredondo Hey just FYI I was told once by @GrabYourPitchforks that it is better to use a constant when calling `stackalloc`: https://github.com/dotnet/runtime/pull/54186#discussion_r661724474

Just passing along the info 😄

Reply:
Hey @CodeBlanch thanks for the info. Yeah I leaved this as a draft just to test if we see some impact by the change. Will revert those and keep only the ones with a constant value to check again, but my hopes are pretty low.